modmesh/app/euler1d.py

```diff
@@ -242,66 +271,12 @@ class PlotConfig():
           configuration table.
         - `_col_header` (list): The header for the configuration
           table columns.
-
-    Methods:
-        - :meth:`data(row, col)`: Get the value at a specific row and column
-          in the configuration table.
-        - :meth:`setData(row, col, value)`: Set the value at a specific row
-          and column in the configuration table.
-        - :meth:`columnHeader(col)`: Get the header for a specific column
-          in the configuration table.
-        - :meth:`editable(row, col)`: Check if a cell in the configuration
-          table is editable.
-        - :meth:`rowCount()`: Get the number of rows in the configuration
-          table.
-        - :meth:`columnCount()`: Get the number of columns in the
-          configuration table.
-        - :meth:`get_var(key, key_row, key_col)`: Get the value of a
-          configuration variable based on its key.
     """
     def __init__(self, input_data):
-        self.state = State(input_data)
-        self._tbl_content = self.state
-        self._col_header = ["variable",
-                            "line_selection",
-                            "y_axis_upper_limit",
-                            "y_axis_bottom_limit"]
-
-    def data(self, row, col):
-        """
-        Get the value at a specific row and column in the configuration table.
-
-        :param row: Row index.
-        :type row: int
-        :param col: Column index.
-        :type col: int
-        :return: The value at the specified location in
-        the configuration table.
-        """
-        return self._tbl_content[row][col]
-
-    def setData(self, row, col, value):
-        """
-        Set the value at a specific row and column in the configuration table.
-
-        :param row: Row index.
-        :type row: int
-        :param col: Column index.
-        :type col: int
-        :prarm value: Any
-        :return None
-        """
-        self._tbl_content[row][col] = value
-
-    def columnHeader(self, col):
-        """
-        Get the specific column header in the configuration table.
-
-        :param col: Column index.
-        :type col: int
-        :return: The header for the specific column.
-        """
-        return self._col_header[col]
+        super().__init__(input_data, ["variable",
+                                      "line_selection",
+                                      "y_axis_upper_limit",
+                                      "y_axis_bottom_limit"])
 
     def editable(self, row, col):
```

Comment:
Since `PlotConfig` table has multiple columns allowing user to modified its value via `QtTableView`. Therefore I overwrite the editable method.